tests/transactions/lease_test.py

```diff
@@ -53,31 +54,31 @@ def expectedV3(self):
             "proofs": ['2BmzCScRy6soyyufzxkNRc3kATCh3HYPtNsGb2Nx6RTkNWXGwMFQLj5cCzKZhJG9TxQHu4DFQyeEuNinJnXC3Ft7']
         }
 
-    def testto_json(self):
+    def test_to_json(self):
         transaction = Lease('3N9ChkxWXqgdWLLErWFrSwjqARB6NtYsvZh', 120000000)
```

Comment:
It doesn't test for both v2 and v2. Use [`@pytest.mark.parametrize`](https://docs.pytest.org/en/6.2.x/parametrize.html)